compiler/control/OMROptions.cpp

```diff
@@ -4614,6 +4658,7 @@ OMR::Options::TR_OptionStringToBit OMR::Options::_optionStringToBitMapping[] = {
 char *
 OMR::Options::setBitsFromStringSet(char *option, void *base, TR::OptionTable *entry)
    {
+   TR_VerboseLog::vlogAcquire();
```

Comment:
Another place where we are acquiring the lock with the possibility of never actually doing a write.